lib/components/normal-components/Testpoint.ts

```diff
@@ -0,0 +1,159 @@
+import { testpointProps } from "@tscircuit/props"
+import { NormalComponent } from "../base-components/NormalComponent/NormalComponent"
+import { Port } from "../primitive-components/Port"
+import type {
+  PcbSmtPad,
+  PcbPlatedHoleCircle,
+  PcbSmtPadRect,
+  PcbSmtPadCircle,
+  SourceSimpleTestPointInput,
+} from "circuit-json"
+import type { BaseSymbolName } from "lib/utils/constants"
+
+export class Testpoint extends NormalComponent<typeof testpointProps> {
```

Comment:
the casing is incorrect in other places, but regardless let's start to fix it before it becomes too much of a problem
```suggestion
export class TestPoint extends NormalComponent<typeof testpointProps> {
```